com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Targets/BuiltInTarget.cs

```diff
@@ -293,14 +341,14 @@ void UpgradeAlphaClip()
                 case PBRMasterNode1 pbrMasterNode:
```

Comment:
You can delete the function TryUpgradeFromMasterNode and the ILegacyTarget interface -- that's probably why we're getting this target pulled into old PBR master graphs when they upgrade.